packages/node/src/main.ts

```diff
@@ -16,4 +16,6 @@ initLogger(
 // Lazy import, to allow logger to be initialised before bootstrap()
 // As bootstrap runs services that requires logger
 const { bootstrap } = require('./init');
-void bootstrap();
+if (!(argv._[0] === 'test')) {
```

Comment:
Whats this change for?

Reply:
the first check was not necessary. Just had to check if the subcommand was `test`

Reply:
I don't think we're guaranteed to have test as the first argument. currently `subql-node -f /path/to/project test` is a valid command but this would fail